### PR TITLE
Move instance variables to search_session

### PR DESCRIPTION
**Pull request by jcoyne**
Also removed `@search_context_response` which was not used.

### CODE REVIEW

File: app/controllers/concerns/blacklight/search_context.rb [hunk 1]
```diff
@@ -111,9 +111,8 @@ def setup_next_and_previous_documents
       response, documents = search_service.previous_and_next_documents_for_search index, search_state.reset(current_search_session.query_params).to_hash
 
       search_session['total'] = response.total
-      @search_context_response = response
-      @previous_document = documents.first
-      @next_document = documents.last
+      search_session['prev'] = documents.first
```
Comment (cbeer) [inline app/controllers/concerns/blacklight/search_context.rb]: I'm a little surprised this works. Do you know if we are creating serialized blobs in the session store every time we do this?

Comment (jcoyne) [reply app/controllers/concerns/blacklight/search_context.rb]: Crap, I didn't realize this was the object in session. I'm going to rework this.

File: app/controllers/concerns/blacklight/search_context.rb [hunk 2]
```diff
@@ -111,9 +111,7 @@ def setup_next_and_previous_documents
       response, documents = search_service.previous_and_next_documents_for_search index, search_state.reset(current_search_session.query_params).to_hash
 
       search_session['total'] = response.total
-      @search_context_response = response
-      @previous_document = documents.first
```
Comment (cbeer) [inline app/controllers/concerns/blacklight/search_context.rb]: I guess we should backport some deprecation for this?

Comment (jcoyne) [reply app/controllers/concerns/blacklight/search_context.rb]: I suppose we could make these deprecated instance variables in a backport.

Comment (cbeer) [reply app/controllers/concerns/blacklight/search_context.rb]: 👍

**Pull request merged**